src/migrations/20201029230535-create-orders.js

```diff
@@ -12,30 +12,12 @@ module.exports = {
         allowNull: false,
         unique: true,
       },
-      userID: {
-        type: Sequelize.INTEGER,
-        references: {
-          model: {
-            tableName: 'Users',
-            schema: 'public',
-          },
-          key: 'id',
-        },
+      userId: {
+        type: Sequelize.UUID,
         allowNull: false,
       },
-      teamID: {
+      teamId: {
```

Comment:
I think this should also be `UUID`, but you may have not done so because Kevin/Hyunzee's PR will be changing it to `STRING` so no worries